src/playback/view/playbacktoolbarmodel.cpp

```diff
@@ -93,7 +93,7 @@ void PlaybackToolBarModel::updateActions()
     MenuItemList settingsItems;
 
     for (const UiAction& action : PlaybackUiActions::settingsActions()) {
-        settingsItems << makeActionWithDescriptionAsTitle(action.code);
```

Comment:
You can get rid of this whole method, since it was not more than an ugly hack just for these two cases...